app/lib/src/pages/article_page.dart

```diff
@@ -8,8 +8,8 @@ import 'package:mondaymorning/src/store/states/article_page/article_page_provide
 import 'package:mondaymorning/src/ui/screens/article_screen.dart';
 import 'package:mondaymorning/src/ui/screens/full_article_loading_screen.dart';
 import 'package:mondaymorning/src/ui/screens/full_error_screen.dart';
-import 'package:mondaymorning/src/ui/screens/full_loading_screen.dart';
 
+@RoutePage<dynamic>(name: 'ArticleRoute')
```

Comment:
Why dynamic? This page doesn't return any data so it should be void instead.


```suggestion
@RoutePage<void>(name: 'ArticleRoute')
```